agents-api/agents_api/activities/execute_system.py

```diff
@@ -6,13 +6,15 @@
 from fastapi.background import BackgroundTasks
```

Comment:
This file was completely refactored so you'd have to make the session.chat changes again. Maybe revert these changes and just merge the other fixes from this PR and start a session.chat one anew?